scalardb-sample/README.md

```diff
@@ -1,162 +1,125 @@
-    ],
-    "secondary-index": [
-      "order_id"
-    ],
-    "columns": {
-      "order_id": "TEXT",
-      "customer_id": "INT",
-      "timestamp": "BIGINT"
-    }
-  },
-  "sample.statements": {
-    "transaction": true,
-    "partition-key": [
-      "order_id"
-    ],
-    "clustering-key": [
-      "item_id"
-    ],
-    "columns": {
-      "order_id": "TEXT",
-      "item_id": "INT",
-      "count": "INT"
-    }
-  },
-  "sample.items": {
-    "transaction": true,
-    "partition-key": [
-      "item_id"
-    ],
-    "columns": {
-      "item_id": "INT",
-      "name": "TEXT",
-      "price": "INT"
-    }
-  }
-}
-```
+The sample e-commerce application shows how users can order and pay for items by using a line of credit.
 
-All the tables are created in the `sample` namespace.
+The database that you will be using in the sample application is Cassandra. Although Cassandra does not provide ACID transaction capabilities, you can make transactions ACID compliant by having your application connect to the database through ScalarDB.
 
-- `sample.customers`: a table that manages customers' information
-    - `credit_limit`: the maximum amount of money a lender will allow each customer to spend when using a credit card
-    - `credit_total`: the amount of money that each customer has already spent by using the credit card
-- `sample.orders`: a table that manages order information
-- `sample.statements`: a table that manages order statement information
-- `sample.items`: a table that manages information of items to be ordered
+{% capture notice--info %}
+**Note**
 
-The Entity Relationship Diagram for the schema is as follows:
+Since the focus of the sample application is to demonstrate using ScalarDB, application-specific error handling, authentication processing, and similar functions are not included in the sample application. For details about exception handling in ScalarDB, see [Handle exceptions](https://github.com/scalar-labs/scalardb/blob/master/docs/api-guide.md#handle-exceptions).
+{% endcapture %}
 
-![ERD](images/ERD.png)
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+### What you can do in this sample application
+
+The sample application supports the following types of transactions:
+
+- Get customer information.
+- Place an order by using a line of credit.
+  - Checks if the cost of the order is below the customer's credit limit.
+  - If the check passes, records the order history and updates the amount the customer has spent.
+- Get order information by order ID.
+- Get order information by customer ID.
+- Make a payment.
+  - Reduces the amount the customer has spent.
+
+## Prerequisites
 
-### Transactions
+- One of the following Java Development Kits (JDKs):
+  - [Oracle JDK 8](https://www.oracle.com/java/technologies/downloads/#java8)
+  - [Oracle JDK 11](https://www.oracle.com/java/technologies/downloads/#java11)
+  - [OpenJDK 8](https://openjdk.org/install/)
+- [Gradle](https://gradle.org/install/)
```

Comment:
When I run `gradle --version` in Terminal, I get `zsh: command not found: gradle`, but I'm still able to run the sample application.

Can we remove Gradle from the list of prerequisites if users don't need to install it? Or does it depend on operating system?

Reply:
Java application projects using Gradle as a package manager usually contain `gradlew` executable file. It's recommended to use local `gradlew` instead of globally installed `gradle` command. So I think we can remove Gradle from the list.

Reply:
@komamitsu Thanks for the explanation! I've remove it from this list and will remove it from the other sample tutorials as well.